lib/backup_restore/s3_backup_store.rb

```diff
@@ -32,9 +32,7 @@ def delete_file(filename)
     end
 
     def download_file(filename, destination_path, failure_message = nil)
-      unless @s3_helper.object(filename).download_file(destination_path)
-        raise failure_message&.to_s || "Failed to download file"
```

Comment:
I'm curious as to why we need to remove this when this is related to the backup store.